ui/app/AppLayouts/Profile/views/MyProfileView.qml

```diff
@@ -166,7 +166,6 @@ SettingsContentBase {
                                                    profileHeader.cropRect.y.toFixed(),
                                                    (profileHeader.cropRect.x + profileHeader.cropRect.width).toFixed(),
                                                    (profileHeader.cropRect.y + profileHeader.cropRect.height).toFixed())
-            reset()
```

Comment:
We can't reset the models because save is async and we don't know when/if the save will be completed.
Now the flow is more natural IMO. We call save with the current state, the backend updates the db and the showcase models after save. Then the `dirty` state is removed in the writable model when the sourceModel is identical to the writable.